src/DynamicMethodTrait.php

```diff
@@ -4,20 +4,147 @@
 
 trait DynamicMethodTrait {
 
-    function __call($fx, $args){
+    public function __call($method, $arguments)
+    {
+        if (($ret = $this->tryCall($method, $arguments))) {
```

Comment:
One brackets here seem not needed -> if ($ret = $this->tryCall($method, $arguments)) {